src/mscorlib/src/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -470,6 +470,27 @@ private void OnProcessExit(object sender, EventArgs e)
 
         // Synchronization primitive for controlling initialization of Default load context
         private static readonly object s_initLock = new Object();
+
+        // Occurs when an Assembly is loaded
+        public static event AssemblyLoadEventHandler AssemblyLoad 
+        { 
```

Comment:
Yes I would expect it to be raised for all contexts as these are at appdomain level. I am adding it to AssemblyLoadContext but will not expose them to public contract. We do not have an AppDomainAugment in corelib ...there wasn't one required.

Yes I have confirmed that load notifications are being raised for LoadFrom apis.